.drone.yml

```diff
@@ -6,7 +6,6 @@ pipeline:
     image: webhippie/hugo:latest
     commands:
       - apk -U add git
-      - git clone https://github.com/go-gitea/theme.git themes/gitea
```

Comment:
For the record: `git submodule foreach git pull origin master` is what you would use if you wanted to set all submodules to their master branch on the origin remote.